### PR TITLE
Update install instructions

### PR DESCRIPTION
**Pull request by tsroten**
This:
- adds a note about Python 2.6
- adds Fedora install instructions
- updates wording on the page

### CODE REVIEW

File: content/pages/1.install.md [hunk 1]
```diff
@@ -39,15 +43,21 @@ The easiest way install ``mycli`` on a Mac is to use [homebrew].
...
+### Fedora
+
+```
+    $ sudo dnf install mycli
+```
+
+### RHEL, Centos:
 
-I haven't built an RPM package for mycli yet. So please use `pip` to install `mycli`. You can install pip on your system using:
+We don't have an RPM package for mycli, yet. So, please use `pip` to install `mycli`. You can install pip on your system using:
```
Comment (amjith) [inline content/pages/1.install.md]: If people are able to install it on Fedora doesn't that mean there is an RPM package for it? 

/cc @meeuw

Comment (tsroten) [inline content/pages/1.install.md]: Good question. Maybe you or @meeuw can help me figure out what better wording for that is.

Is there an easy, non-pip way to install mycli on RHEL or Centos?

If not, is it better to simply say "We don't have packages for RHEL or Centos, yet. So, please use pip..."?

Comment (meeuw) [inline content/pages/1.install.md]: The RPM can be used on Centos/RHEL but most of the dependencies are missing. I've created a repository with all the dependencies on https://copr.fedorainfracloud.org/coprs/meeuw/mycli-epel/

I'd like to get this into Fedora-EPEL but haven't found the time to do this.

Comment (tsroten) [inline content/pages/1.install.md]: @meeuw Great, no problem.

So, for now, does this sound ok? "We don't have packages for RHEL or Centos, yet. Instead, use pip..."

@amjith

**Comment by tsroten** [comment]
@meeuw - Thanks for being so thorough. Your comment is there, you have to click on "Show Outdated", though.

I reworded the docs in this pull request. Then, whenever it changes in the future, we can update it then. Thanks!

**Comment by meeuw** [comment]
:+1: This is fine for now

File: content/pages/1.install.md [hunk 2]
```diff
@@ -34,24 +38,27 @@ The easiest way install ``mycli`` on a Mac is to use [homebrew].
...
-```
+    :::bash
     $ sudo apt-get update
     $ sudo apt-get install mycli
-```
 
-### RHEL, Centos, Fedora:
+### Fedora
+
+    :::bash
```
Comment (amjith) [inline content/pages/1.install.md]: This triple colon is not rendering properly in md. 

Is that intentional?

Comment (tsroten) [inline content/pages/1.install.md]: Hmm, yeah, I think that's how Pelican does code blocks. Github-flavored Markdown is different.

To be honest, I was imitating the triple-colons that were already in the file. Looking at [Pelican's docs](http://docs.getpelican.com/en/3.6.3/content.html#syntax-highlighting), it seem like what they recommend (and when you run the development server, it renders).

screen shot 2017-06-02 at 21 18 16

**Comment by amjith** [comment]
:fox_face:

**Pull request merged**